config/routes.rb

```diff
@@ -4,5 +4,8 @@
   scope "(:locale)", locale: /en|vi/ do
     get "static_pages/home"
     get "static_pages/help"
+    get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
+    get "/users/:id", to: "users#show"
```

Comment:
```suggestion
    resources :users, only: :show
```
ri cụ
dùng của rails support nhá